godel_simple_gui/src/blending_widget.cpp

```diff
@@ -159,13 +159,27 @@ void godel_simple_gui::BlendingWidget::setLabelText(const std::string& txt)
 std::vector<std::string> godel_simple_gui::BlendingWidget::getPlanNames()
 {
   std::vector<std::string> selected_plans;
-  if (ui_->plan_list_widget->currentItem() == NULL)
-    ROS_WARN_STREAM("No plan slected, TODO QERRORMESSAGE");
+  if (planSelectionEmpty())
+    ROS_ERROR_STREAM("No plan slected, TODO QERRORMESSAGE");
   selected_plans.push_back(ui_->plan_list_widget->currentItem()->text().toStdString());
   return selected_plans;
 }
 
-void godel_simple_gui::BlendingWidget::sendGoal(godel_msgs::SelectMotionPlanActionGoal goal)
+void godel_simple_gui::BlendingWidget::sendGoal(const godel_msgs::SelectMotionPlanActionGoal goal)
```

Comment:
This should be passed by reference, it's missing the **&** symbol on the input argument.